src/pocof/Query.fs

```diff
@@ -135,6 +135,22 @@ module PocofQuery =
           Is = is
           Answer = answer }
 
+    type InternalState with // TODO: There might be a better way.
+        static member prepareNotification state =
+            { state with Notification = prepareNotification state }
```

Comment:
I think it can replace with `module InternalState = ~` because there is no name duplication.